wgvanitykey.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python
+"""
+wgvanitykey -- generate Curve25519 sk/pk keypairs
+and search for a given string in the base64 encoding of the public key
+
+Usage::
+
+    wgvanitykey -h
+    wgvanitykey -c 1 test  # search for a pk that starts with 'test'
+    wgvanitykey -c 1 test -m contains  # search for a pk that contains 'test'
+"""
+import functools
+import logging
+import sys
+import unittest
+from base64 import b64encode
+from multiprocessing import Pool, Process, cpu_count, Value, Queue
+from string import ascii_uppercase, ascii_lowercase, digits
+
+from nacl.public import PrivateKey
+
+version = "0.1.0"
```

Comment:
bump2version/bumpversion (included with cookiecutter-pypackage (see #3)) can update this version automatically.
- https://pypi.org/project/bump2version/
  - https://github.com/c4urself/bump2version
- https://pypi.org/project/bumpversion/
  - https://github.com/peritus/bumpversion
- https://github.com/audreyr/cookiecutter-pypackage